src/components/callout/callout.tsx

```diff
@@ -1,9 +1,11 @@
 import React, { ReactNode } from 'react';
 import styled from 'styled-components';
 import { up } from '../breakpoint/breakpoint';
+import { Illustration } from '../illustration/illustration';
+import { IllustrationType } from '../illustration/illustration-set';
 
-interface CalloutProps {
-  illustration?: React.ReactNode;
+export interface CalloutProps {
```

Comment:
Why did you decide to export this?

Reply:
One desperate attempt to render the docs correctly. I though the extractor needs this to be exported (having some feeling from my experiences with the introspection utility in our customer project). I will revert this of course, sorry this slipped in.